app/gui/src/dashboard/layouts/AssetPanel/components/AssetProperties.tsx

```diff
@@ -28,11 +28,20 @@ import { useFullUserSession } from '#/providers/AuthProvider'
 import { useFeatureFlags } from '#/providers/FeatureFlagsProvider'
 import { useText } from '#/providers/TextProvider'
 import type Backend from '#/services/Backend'
-import { AssetType, BackendType, Plan, type AnyAsset, type DatalinkId } from '#/services/Backend'
+import {
+  AssetType,
+  BackendType,
+  getAssetPermissionId,
+  getAssetPermissionName,
+  Plan,
+  type AnyAsset,
+  type DatalinkId,
+} from '#/services/Backend'
 import * as permissions from '#/utilities/permissions'
 import { tv } from '#/utilities/tailwindVariants'
 import { useStore } from '#/utilities/zustand'
 import { useMutation } from '@tanstack/react-query'
+import { toReadableIsoString } from 'enso-common/src/utilities/data/dateTime'
```

Comment:
Why we don't just use the `toLocaleString()` instead?

Reply:
i think we prefer a fixed format to minimize confusion. @jdunkerley thoughts?

i know the "modified at" column in Figma uses a slightly different custom format (but very similar to ISO) - should we remove the custom format completely in favor of ISO?

or should we perhaps add this as a setting (now or in the future) to let the user specify date-time format, but still default to ISO?

Reply:
I think it should be defined by user locale. 

In the Asset Version list I use this format: ```typescript         
 {new Date(version.lastModified).toLocaleString(locale, {
                year: 'numeric',
                month: 'short',
                day: 'numeric',
                hour: 'numeric',
                minute: 'numeric',
              })}
```

Reply:
well, it was user locale for the scheduler, and i was requested to change it to ISO, which is why I'm not sure

Reply:
ISO form is great.